### DISCUSSION TITLE
How is EMPERL_PRELOAD_FILE or PRELOAD_FILE used?

### DISCUSSION THREAD

**Discussion by vadimkantorov** [haukex/webperl#22] [Q&A]
I'm trying to understand what files/directories get bundled with the WASM module, but I can't find how/where this variables from https://github.com/haukex/webperl/blob/master/build/emperl_config.sh get used

**Comment by haukex** [comment]
Note WebPerl currently consists of two repositories, this one plus the patched Perl5 source. `EMPERL_PRELOAD_FILE` is used for `emcc`'s `--preload-file` option, see https://github.com/haukex/emperl5/blob/327812bab/Makefile.SH#L1097
